pkg/config/autodiscovery/autodiscovery.go

```diff
@@ -33,6 +33,12 @@ func DiscoverComponentsFromConfig() ([]config.ConfigurationProviders, []config.L
 		detectedProviders = append(detectedProviders, prometheusProvider)
 	}
 
+	// Add database-monitoring aurora listener if the feature is enabled
+	if config.Datadog.GetBool("database_monitoring.autodiscovery.aurora.enabled") {
+		detectedListeners = append(detectedListeners, config.Listeners{Name: "_dbm_aws_aurora"})
```

Comment:
The listener name should not be `_` unlike internal AD identifiers.
```suggestion
		detectedListeners = append(detectedListeners, config.Listeners{Name: "dbm_aws_aurora"})
```